lib/internal/process/modules.js

```diff
@@ -10,6 +10,10 @@ const { getURLFromFilePath } = require('internal/url');
 const Loader = require('internal/loader/Loader');
 const path = require('path');
 const { URL } = require('url');
+const {
+  initImportMetaMap,
+  wrapToModuleMap
+} = require('internal/vm/Module-common');
```

Comment:
you can just export the map from `internal/vm/module`, and if for some reason this file is kept please rename it to `module_common.js` to keep with the rest of our files.

Reply:
I agree, I can move the maps to Module.js.